docs/general/networking/index.md

```diff
@@ -15,7 +15,8 @@ HTTP and HTTPS are the primary means of connecting to the server. If using a sel
 
 :::caution
 
-In order for Chromecast to work on a non-public routable connection, 8.8.8.8 must be blocked on the Chromecast's Gateway. Blocking 8.8.8.8 on your router is the easiest solution to this problem.
+In order for Chromecast to work on your local LAN, the easiest solution is to use IPv6 instead of IPv4. For a public routable IPv6 (not a link-local or ULA) there is no differentiation between public or local. An IPv6 address can simultaneously be publicly routable and accessible from the local LAN.
```

Comment:
Will it work if I have a link local ipv6? (One that starts with fe80)

Reply:
I have not tested this but I think no. Chromecast needs a valid none self signed cert last time I checked.  Since you can't create a fe80 (ULA) AAAA record with a valid lets encrypt cert, the IPv6 has to be public routable. 

That is why I included the hint in the brackets.